### PR TITLE
WIP: v9 catalog bump

### PR DESCRIPTION
**Pull request by rhoads-zach**
@inoxx03 @tradej please merge and test on Monday and coordinate with @gastaldi

### CODE REVIEW

**Comment by gastaldi** [comment]
Ah wait, need to bump the openshift templates too. That should take some mins

**Comment by rhoads-zach** [comment]
Thank you @gastaldi.

@tradej @inoxx03 feel free to pick this up Monday AM

**Comment by gastaldi** [comment]
All good now ;)

File: nginx.conf [hunk 1]
```diff
@@ -53,7 +53,7 @@ http {
         }
 
         location /latest-launcher-template {
-            return 301 https://raw.githubusercontent.com/openshiftio/launchpad-templates/v8/openshift/launchpad-template.yaml;
+            return 301 https://raw.githubusercontent.com/openshiftio/launchpad-templates/v9/openshift/launchpad-template.yaml;
```
Comment (tradej) [inline nginx.conf]: Am I correct to assume this line will always have to be updated with the new release? If so, this needs a mention in the Contrib Guide.

**Comment by inoxx03** [comment]
merging

**Comment by inoxx03** [comment]
merged: 5935f4309e3631c0f7921d6d94129e2f362ab134

**Pull request closed without merging**